libnavigation-core/src/main/java/com/mapbox/navigation/core/trip/session/MapboxTripSession.kt

```diff
@@ -363,7 +360,7 @@ internal class MapboxTripSession(
     ) {
         voiceInstructionsObservers.add(voiceInstructionsObserver)
         routeProgress?.let {
```

Comment:
This check isn't needed anymore, is it?

Reply:
It's unlikely to be `true` but technically feasible. I removed it in the first place and then thought about it and decided it to leave it as we had it before.